<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>334</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Окно Регистрация</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
.QLabel
{
	background-color: rgb(0, 0, 127);
	color: rgb(255, 255, 255);
}
.QPushButton
{
   
	background-color: rgb(42, 173, 255);
	color: rgb(0, 0, 0);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 11px;
    min-width: 8em;
    padding: 4px;
}
.QPushButton:hover
{
   border-style: outset;
    border-width: 3px;
    border-radius: 12px;
	background-color: rgb(255, 255, 255);
}
.QPushButton:pressed
{
background-color: rgb(255, 0, 0);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="snameEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>E mail</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mailEdit">
       <property name="placeholderText">
        <string>example@bk.ru</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Парол</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="pasEdit">
       <property name="inputMask">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Enter Password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Создать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
